docs/source-app/workflows/byoc/index.rst

```diff
@@ -5,62 +5,287 @@ Run Apps on your own cloud (BYOC)
+
+Once IT or you yourself have created the role, you need to get the ARN and external ID from the AWS console.
+
+Here's a short video showing you the process.
+
+.. raw:: html
+
+   <video id="background-video" autoplay loop muted controls poster="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/aws_signin.png" width="100%">
+      <source src="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/byoc_aws_get_arn_exid.mp4" type="video/mp4" width="100%">
+   </video>
 
-Next, as part of the environment creation process, you can configure networking,
-security, and select among cluster configuration options based on their own use cases.
+----
+
+*************************
+Step 2: Install Lightning
+*************************
+
+Lightning must be installed before you can create a Lightning BYOC cluster.
+
+----
+
+Who should install Lightning?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once your AWS account is ready, IT’s job is done. That means you are the one who will be doing this (researchers, data scientists, Lightning App creators).
+
+----
+
+Install Lightning
+^^^^^^^^^^^^^^^^^
+
+From your local machine use the following command to install Lightning:
+
+.. code:: bash
+
+    python -m pip install lightning
+
+We have versioned pinned requirements at the time of the writing. Feel free to use the newer versions:
+
+.. code:: bash
+
+    torch==1.1
+    torchvision
+    pytorch_lightning
+
+----
+
+******************************************
+Step 3: Set up your Lightning BYOC cluster
+******************************************
+
+Creating a cluster is a one time operation, unless you intend to add more clusters.
+
+----
 
-After submitting a cluster creation request, the Lightning Control Plane creates the required cloud infrastructure on the user account. This
-sets up a new Lightning Cluster along with a Lightning Kubernetes Operator.
+Who should set up your Lightning BYOC cluster?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+With Lightning installed, researchers, data scientists, Lightning App creators, and more, can now start creating and monitoring clusters.
+
+----
 
-*******************************
 Create a Lightning BYOC cluster
-*******************************
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You must have your cloud configured before you try and create a BYOC cluster.
 
-And to make your life a little easier, we've made a `Terraform module to help with that <https://github.com/Lightning-AI/terraform-aws-lightning-byoc>`_.
-
-Create a Lightning BYOC cluster using the following command:
+Create a Lightning BYOC cluster using the following command from your local machine:
 
 .. code:: bash
 
-   lightning create cluster <cluster-name> <cloud-provider-parameters>
+     python -m lightning create cluster <cluster-name> --provider aws --role-arn <ARN-you-copied-earlier> --external-id <external-id-you-copied-earlier> --region <aws-region> <other-cloud-provider-parameters> --wait
```

Comment:
why is there python -m in front?
The standard use is without. The CLI interaction should always be `lightning x y z`

Reply:
@awaelchli

The dev team tells me this is required. @nmiculinic please verify.